src/main/kotlin/InputView.kt

```diff
@@ -0,0 +1,5 @@
+object InputView {
```

Comment:
패키지를 만들어서 전체 프로그램을 위치시키면 어떨까요? 학습 테스트 등을 만들 때 별도의 패키지로 분리할 필요가 있을 수 있습니다.
```suggestion
package minesweeper

object InputView {
```

Reply:
fee5c861e87d02ea7a79b485899422a99c8d0576 반영하였습니다